WebsiteChecker.py

```diff
@@ -45,36 +45,41 @@
     # First test, send get request if get error or the srv take more then 30 second to response - Fail
     start = time.time()
     try:
-        request = requests.get(site, headers=header)
-        if request.status_code == 200:
-            customFunc.emailfunc.reset_error_counter('CodeError', service, site)
+        for failure_counter in range(0, 3):

```

Comment:
how many time would this loop run?
I think 4
I think you meant to write:
```suggestion
        for failure_counter in range(1, 3):

```

Reply:
It runs 3 times as expected,
The example I used was (0, 5) - to run 5 times.
So here it's like i wrote it from what i saw in the breakpoint